holidays/countries/pakistan.py

```diff
@@ -22,7 +22,11 @@ class Pakistan(HolidayBase, InternationalHolidays, IslamicHolidays):
 
     def __init__(self, *args, **kwargs):
         InternationalHolidays.__init__(self)
-        IslamicHolidays.__init__(self, cls=PakistanIslamicHolidays)
+        IslamicHolidays.__init__(
+            self,
+            cls=PakistanIslamicHolidays,
+            show_estimated=kwargs.pop("islamic_show_estimated", True),
```

Comment:
maybe
```suggestion
            show_estimated=kwargs.pop("show_estimated_islamic_holidays", True),
```

Reply:
"show_estimated_islamic_holidays" - that name makes us think that this param controls the actual showing of such holidays, not just their visual appearance. 🙄